docs/automated-testing/ui-testing/README.md

```diff
@@ -61,8 +61,11 @@ Recommended testing frameworks:
   - [Cypress](https://www.cypress.io)
   - [Jest](https://jestjs.io/docs/en/snapshot-testing)
   - [Selenium](https://www.selenium.dev)
+  - [Appium](https://appium.io)
 - OS/Mobile Applications
   - [Coded UI tests (CUITs)](https://learn.microsoft.com/en-us/visualstudio/test/use-ui-automation-to-test-your-code?view=vs-2019)
   - [Xamarin.UITest](https://learn.microsoft.com/en-us/appcenter/test-cloud/uitest/)
+  - [BrowserStack](https://www.browserstack.com)
+  - [Appium](https://appium.io)
```

Comment:
Too many Appiums! Lose one :)

Reply:
Hmm, the intention is to add Appium as a valid option for web AND for OS/Mobile (similar to BrowserStack, which works for both). Hence I listed it twice.

Reply:
Sorry, my mistake. I didn't notice the nesting.